ipcsample/src/main/java/com/mateuszkoslacz/moviper/ipcsample/viper/presenter/MainPresenter.java

```diff
@@ -0,0 +1,84 @@
+package com.mateuszkoslacz.moviper.ipcsample.viper.presenter;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import com.mateuszkoslacz.moviper.base.presenter.ViperActivityBaseRxPresenter;
+import com.mateuszkoslacz.moviper.ipcsample.viper.contract.MainContract;
+import com.mateuszkoslacz.moviper.ipcsample.viper.interactor.MainInteractor;
+import com.mateuszkoslacz.moviper.ipcsample.viper.routing.MainRouting;
+import com.mateuszkoslacz.moviper.ipcsample.viper.view.fragment.ViperFragment;
+
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_BLUE;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_GREEN;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_NAME_BLUE;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_NAME_GREEN;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_NAME_RED;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.COLOR_RED;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.FRAGMENT_NAME_FIRST;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.FRAGMENT_NAME_SECOND;
+import static com.mateuszkoslacz.moviper.ipcsample.util.Constants.FRAGMENT_NAME_THIRD;
+
+public class MainPresenter
+        extends ViperActivityBaseRxPresenter
+        <MainContract.View,
+                MainContract.Interactor,
+                MainContract.Routing>
+        implements
+        MainContract.Presenter {
+
+    public MainPresenter(Activity activity) {
+        super(activity);
+    }
+
+    @Override
+    public void onViewCreated() {
+        if (isViewAttached()) {
+            createAndConfigureFirstFragment();
+            createAndConfigureSecondFragment();
+            createAndConfigureThirdFragment();
+        }
+    }
+
+    private void createAndConfigureFirstFragment() {
+        ViperFragment viperFragment = new ViperFragment();
+        viperFragment.setArguments(prepareBundleToViperFragment(FRAGMENT_NAME_FIRST,
+                COLOR_NAME_BLUE, COLOR_BLUE));
+        getRouting().bindFragmentToView(getView().getViewIdFragmentFirst(), viperFragment);
+    }
+
+    private void createAndConfigureSecondFragment() {
```

Comment:
All of these 3 methods should have extracted common code — all of them are almost the same.